htdocs/user/class/user.class.php

```diff
@@ -1419,7 +1419,7 @@ public function create_from_contact($contact, $login = '', $password = '')
 		if ($result > 0) {
 			$sql = "UPDATE ".MAIN_DB_PREFIX."user";
 			$sql .= " SET fk_socpeople=".$contact->id;
-			$sql .= ", civility=".$contact->civility_code;
+			$sql .= ", civility=\"".$contact->civility_code."\"";
```

Comment:
For SQL, the string char to use is '  (simple quote).  
Note, it is " (double quotes) for html tag content.